mesa/Person.py

```diff
@@ -156,6 +156,15 @@ def makeParent_OffspringsLinks(self, asker):
             self.neighbors.get("offspring").add(asker)
             asker.neighbors.get("parent").add(self)
 
+    def makeSchoolLinks(self, asker):
```

Comment:
What about avoiding repeated lines and going with something like the snippet here?
```
        if not isinstance(asker, Iterable): asker = [asker]
        for person in asker:
                self.neighbors.get("school").add(person)
                person.neighbors.get("school").add(self)
```

PS: does it ever get called from a person?

Reply:
The proposed solution is better. But at the moment it is never called this procedure for two individual agents, so I will eliminate the condition.